pages/workspace/workspace.ejs

```diff
@@ -18,7 +18,10 @@
     <div class="d-flex flex-row ml-auto align-items-center">
       <ul class="navbar-nav flex-row">
         <li class="mr-2">
-          <span id="state" class="badge badge-dark text-uppercase"></span>
+          <div class="badge badge-dark">
+            <span id="state" class="text-uppercase"><i class="fas fa-spinner fa-pulse"></i></span>
+            <span id="message" class="font-weight-normal"></span>
```

Comment:
It looks to me like the state and message are too close together?

Reply:
You mean the visual rendering? As in you want more separation between the state and message?

Reply:
Yep, or an emdash or colon or something? You are better at visual design than me, but it just looked a bit squashed.

Reply:
Tried a slight redesign (updated OP screenshot)

Reply:
I really like the new status/message badge! Looks great.

When the display width is narrow the status badge is right aligned, which causes the status (e.g., "launching") to jump around a lot as the message changes. It also makes a hamburger menu that displays the reboot/help buttons in a weird overlay way (doesn't look like a menu). I'm not sure whether this is all intentional?

Reply:
Not intentional, but I haven't figured out how to fix it. I was kinda just hoping no one would be using workspaces in a mobile viewport 😆 but I can dig into it more if you want.

Reply:
One workaround could be to just hide the `message` part in the `xs` breakpoint.

Reply:
I think it's fine, I just wasn't sure whether it's intentional. Your screenshot in the PR shows the `xs` view, which is why I thought you might have intended this 😄